src/main/resources/assets/viafabricplus/lang/en_us.json

```diff
@@ -83,6 +83,7 @@
   "visual_settings.viafabricplus.enable_block_hit_animation": "Enable block hit animation",
   "visual_settings.viafabricplus.disable_server_pinging": "Disable server pinging",
   "visual_settings.viafabricplus.sideways_backwards_walking": "Sideways backwards walking",
+  "visual_settings.viafabricplus.enable_legacy_tablist": "Enable legacy tablist behavior",
```

Comment:
translation key / setting field should have the same name as the translation

Reply:
the translation key is `enable_legacy_tablist` and the field is `enableLegacyTablist`, do you mean the actual translation text `"Enable legacy tablist behavior"` should be plainly `"Enable legacy tablist"` ?